### ISSUE TITLE
Have some kind of indicator for scripts that download 3rd party resources

### ISSUE THREAD

**Issue by silversword411** [amidaware/tacticalrmm#1044]
We were discussing scripts that do downloads of other non-tracked programs (like https://github.com/amidaware/community-scripts/blob/main/scripts/Win_Network_Speed_Test.ps1 )

Should we have some kind of indictor (*, colorize it differently in the library etc) so it'll queue people to double-check it.

**Comment by dinger1986** [comment]
I was thinking an `!`

Also if we could add a way to download testing scripts as well?

**Comment by bbrendon** [comment]
maybe have a value in the script json database that triggers a GUI element.

**Comment by sadnub** [comment]
Could we just search the script body and args for `[protocol]://` and automatically return to the UI that the script references external resources? This would make it automatic and keep maintenance down.

**Comment by dinger1986** [comment]
Absolutely, whatever works easiest. Think the main thing is agreeing on an indicator
